### PR TITLE
Wrap up licensing subproject

### PR DESCRIPTION
**Pull request by dims**
The dependencies for kubernetes/kubernetes is in good shape, we can handle any items left as part of area/code-organization (sig-arch) if needed.

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

**Comment by dims** [comment]
cc @justaugustus @swinslow @nikhita

**Review by nikhita — APPROVED** [review]
/lgtm

/hold 
for https://github.com/kubernetes/sig-release/pull/1412 to merge

**Review by hasheddan — APPROVED** [review]
/lgtm

File: label_sync/labels.yaml
Comment (nikhita) [inline label_sync/labels.yaml:21]: /lgtm cancel

we need a `deleteAfter` field here - https://github.com/kubernetes/test-infra/tree/master/label_sync#configuration

Comment (dims) [reply label_sync/labels.yaml:21]: Done!

**Comment by dims** [comment]
@nikhita can you please check if this is right now?

**Review by spiffxp — CHANGES_REQUESTED** [review]
```ERROR: labels.md out of date. Fix with hack/update-labels.sh```

**Comment by dims** [comment]
rebased. thanks @spiffxp

**Comment by spiffxp** [comment]
hm... it's still complaining about the same thing, maybe the delete date being in the past is an issue?

**Comment by dims** [comment]
/hold cancel

( since https://github.com/kubernetes/sig-release/pull/1412 is merged)

@spiffxp @nikhita please `/lgtm` if appropriate

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

**Pull request merged**
